mongodbatlas/data_source_mongodbatlas_third_party_integration.go

```diff
@@ -37,11 +37,6 @@ func thirdPartyIntegrationSchema() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"license_key": {
-				Type:      schema.TypeString,
-				Sensitive: true,
-				Computed:  true,
-			},
```

Comment:
Are you sure that NEW_RELIC is deprecated? It is still available in our Public DOC: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Third-Party-Integrations/operation/createThirdPartyIntegration

Reply:
yes, it was deprecated along with flowdock integration in v1.10.0. You can check it [in this release PR](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1225/files#diff-8e65feb4edd9f3cade8198fe8f62bcbb4dfb59ce8ec2ea22f8c31ba7f7ad04b8L34) in `mongodbatlas/resource_mongodbatlas_third_party_integration.go` file. But for some reason docs were not updated when it was deprecated

Reply:
Thanks. I found this ticket https://jira.mongodb.org/browse/INTMDB-408 with additional details